CTOR-924 : Refresh network-routers-atrica-snmp (#3624)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-atrica-snmp.md

id: network-routers-atrica-snmp
title: Atrica Routeur
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Contenu du pack

### Modèles

Le connecteur de supervision **Atrica Routeur** apporte un modèle d'hôte :

* **Net-Atrica-SNMP-custom**

Le connecteur apporte les modèles de service suivants
(classés selon le modèle d'hôte auquel ils sont rattachés) :

<Tabs groupId="sync">
<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">

Pas de modèle de service rattaché à ce modèle d'hôte.

> 
> > Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Atrica-SNMP-custom** est utilisé.

Author: ```suggestion
```

| CRITICALOUTEIRDISCARD | Thresholds                                                                                                                                       |                    |             |
| EXTRAOPTIONS          | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                    |             |

</TabItem>
<TabItem value="Connections-Global" label="Connections-Global">

| Macro                 | Description                                                                                                                                                                                                                                                                            | Valeur par défaut  | Obligatoire |
|:----------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|

> | FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                                                                                                                                                                              | .+                 |             |

Author: ```suggestion
| FILTER                | Set the interface (number expected) example: 1,2,... (empty means 'check all connections'). Can be a regex                                                                                                                                                                                              | .+                 |             |
```

id: network-routers-atrica-snmp
title: Atrica Routeur
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Contenu du pack

### Modèles

Le connecteur de supervision **Atrica Routeur** apporte un modèle d'hôte :

* **Net-Atrica-SNMP-custom**

Le connecteur apporte les modèles de service suivants
(classés selon le modèle d'hôte auquel ils sont rattachés) :

<Tabs groupId="sync">
<TabItem value="Net-Atrica-SNMP-custom" label="Net-Atrica-SNMP-custom">

| Alias | Modèle de service | Description |

> |:------|:------------------|:------------|

Reviewer: nothing

> | Alias | Modèle de service | Description |
> |:------|:------------------|:------------|

Author: ```suggestion
Pas de modèle de service rattaché à ce modèle d'hôte.
```

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Atrica-SNMP-custom** est utilisé.

</TabItem>
<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">

| Alias                    | Modèle de service                               | Description          |
|:-------------------------|:------------------------------------------------|:---------------------|

> | Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le traffic  |

Reviewer: ```suggestion
| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic  |
```

> | Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le traffic  |

Reviewer: ```suggestion
| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic  |
```

> | Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le traffic  |

Reviewer: we have 3 times the same description for 3 different things

> | Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic  |
> | Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic  |
> | Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |

Author: ```suggestion
| Connections-Generic-Id   | Net-Atrica-Connections-Generic-Id-SNMP-custom   | Contrôle le trafic de l'équipement ayant un ID spécifié |
| Connections-Generic-Name | Net-Atrica-Connections-Generic-Name-SNMP-custom | Contrôle le trafic de l'équipement ayant un nom spécifié |
| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |
```

> | Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le traffic  |

Reviewer: ```suggestion
| Connections-Global       | Net-Atrica-Connections-Global-SNMP-custom       | Contrôle le trafic  |
```
